k8s/conjur/chart/conjur/templates/config/secret-config-envs.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}-config-envs
+  labels:
+    app.kubernetes.io/name: "{{ .Release.Name }}"
+    app.kubernetes.io/component: conjur
+data:
+  POSTGRES_PASSWORD: {{ .Values.conjur.db.password | b64enc | quote }}
+  CONJUR_DATA_KEY: {{ .Values.conjur.data_key | b64enc | quote }}
+  DATABASE_URL: {{ printf "postgres://conjur:%s@%s-postgresql-svc/conjur" .Values.conjur.db.password .Release.Name | b64enc | quote }}
+
```

Comment:
Remove extralines

Reply:
blank line at the end is a best practice, 3 blank lines - best-best-best practice, but i'll fix it